docs/source/reference/graphos-features.mdx

```diff
@@ -1,41 +1,41 @@
 ---
 title: GraphOS Router Features
-subtitle: Use router features enabled by GraphOS and the Enterprise plan
-description: Unlock Enterprise features for the GraphOS Router by connecting it to Apollo GraphOS.
+subtitle: Use router features enabled by GraphOS and licensed plans
+description: Unlock additional features for the GraphOS Router by connecting it to Apollo GraphOS.
 redirectFrom: 
   - /router/enterprise-features
 ---
 
-A router connected to GraphOS, whether cloud-hosted or self-hosted, is called a **GraphOS Router**. It has access to specific GraphOS features depending on the connected GraphOS organization's plan. Refer to the [pricing page](https://www.apollographql.com/pricing#graphos-router) to compare GraphOS Router features across plan types.
+A router connected to GraphOS, whether cloud-hosted or self-hosted, is called a **GraphOS Router**. It has access to specific GraphOS features depending on the connected GraphOS organization's licensed plan. Refer to the [pricing page](https://www.apollographql.com/pricing#graphos-router) to compare GraphOS Router features across plan types.
 
 ## GraphOS Router features
 
 <GraphOSRouterFeatures />
 
 For details on these features, see [this blog post](https://blog.apollographql.com/apollo-router-v1-12-improved-router-security-performance-and-extensibility) in addition to the documentation links above.
 
-## Enterprise plan features
+## Licensed plan features
 
 <Tip>
 
-Try the Enterprise features of GraphOS Router with a free [GraphOS trial](https://www.apollographql.com/pricing).
+Try the licensed features of GraphOS Router with a free [GraphOS trial](https://www.apollographql.com/pricing).
 
 </Tip>
 
-To enable support for Enterprise features in GraphOS Router:
+To enable support for licensed features in GraphOS Router:
 
-- Your organization must have a [GraphOS Enterprise plan](https://www.apollographql.com/pricing/).
+- Your organization must have a [GraphOS licensed plan](https://www.apollographql.com/pricing/).
```

Comment:
I don't think the pricing page clearly establishes with a "licensed plan" is. For example, is the Free plan a licensed plan?